channels/co.m3u

```diff
@@ -63,3 +65,9 @@ http://api.new.livestream.com/accounts/28238881/events/8574038/live.m3u8
 https://live-edge-rbx-1.cdn.enetres.net/184784E1D210401F8041E3E1266822CC021/live-1500/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/kady79t.jpg" group-title="",UAO Play +
 https://live-edge-sxb-1.cdn.enetres.net/184784E1D210401F8041E3E1266822CC023/live-1500/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/q18q9bC.jpg" group-title="Sport",WIN Sports 
+http://ummodigital.com:8000/jorge1/jorge1/4893
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/q18q9bC.jpg" group-title="Sport",WIN Sports 
+http://ummodigital.com:8000/jorge1/jorge1/2292
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/q18q9bC.jpg" group-title="Sport",WIN Sports 
+http://ummodigital.com:8000/jorge1/jorge1/9114
```

Comment:
All 3 links do not work

Reply:
Strange, In Vlc they work when i open them

Reply:
Really strange. VLC gives me only errors for all 3 links, and if I look at the response code there 401 Unauthorized.